process/transactions/datafield/parser.go

```diff
@@ -174,6 +174,10 @@ func parseBlockingOperationESDT(args [][]byte, funcName string) *ResponseParseDa
 	}
 
 	token, nonce := extractTokenIdentifierAndNonce(args[0])
+	if !isASCIIString(string(token)) {
```

Comment:
the ASCII check is too large for token identifiers. Token identifiers can only contain:
- `a..z`
- `A..Z`
- `0..9`
- `-`

cand you please change all the checks done for tokens ?

Reply:
I will let it as is because the protocol does not check these conditions for a token. In case you have an invalid ESDTTransfer with an invalid `token` in arguments you will get `insufficient balance` error. With this check, I ensure that will not have strange strings in `Elasticsearch` db